chatkit/src/main/java/com/stfalcon/chatkit/messages/MessageTextUtils.kt

```diff
@@ -150,5 +197,9 @@ class MessageTextUtils {
             return label ?: content
         }
     }
+
+    enum class MarkDown{
```

Comment:
we dont like enums ;) use @StringDef annotation https://developer.android.com/reference/android/support/annotation/StringDef.html

enums are heavier in memory